labgrid/driver/networkusbstoragedriver.py

```diff
@@ -93,7 +93,8 @@ def write_image(self, filename=None, mode=Mode.DD, partition=None, skip=0, seek=
                 "bs={}".format(block_size),
                 "skip={}".format(skip),
                 "seek={}".format(seek),
-                "conv=fdatasync"
+                "conv=fdatasync",
+                "oflag=dsync"
```

Comment:
What's the performance impact when combined with the bs=512 change that is already merged?

Reply:
It's pretty bad, dd'ing 380 MB takes 3741s (with `oflag=dsync`) vs. 118.966s (without). Should we drop `oflag=dsync` `if skip or seek`? I personally use skip and seek for bootloader replacement only (while keeping the MBR).

Reply:
I'll drop the oflag=dsync patch.